bgpd/bgp_vty.c

```diff
@@ -6116,11 +6131,11 @@ ALIAS_HIDDEN(no_neighbor_addpath_tx_bestpath_per_as,
 	     "Use addpath to advertise the bestpath per each neighboring AS\n")
 
 DEFUN_NOSH (address_family_ipv4_safi,
-       address_family_ipv4_safi_cmd,
-       "address-family ipv4 [<unicast|multicast|vpn|labeled-unicast>]",
-       "Enter Address Family command mode\n"
-       "Address Family\n"
-       BGP_SAFI_WITH_LABEL_HELP_STR)
+	address_family_ipv4_safi_cmd,
```

Comment:
Why is space/tab getting changed here? Same for others.

Reply:
checkstyle.